Simplenote/SPNoteCellView.m

```diff
@@ -39,26 +41,27 @@ - (void)setBackgroundStyle:(NSBackgroundStyle)backgroundStyle
     // because it's also being done via a binding. But ensures all fonts and colors are set appropriately.
     if (self.superview && [self.superview isKindOfClass:[NSTableRowView class]]) {
         NSTableRowView *row = (NSTableRowView*)self.superview;
-        highlighted = row.isSelected;
+        _highlighted = row.isSelected;
     }
 
     [self updatePreview];
 }
 
 - (void)updatePreview
 {
-    NSColor *headlineColor = highlighted ? [self.theme colorForKey:@"tintColor"] : [self.theme colorForKey:@"noteHeadlineFontColor"];
-    NSColor *previewColor = highlighted ? [self.theme colorForKey:@"tintColor"] : [self.theme colorForKey:@"noteBodyFontPreviewColor"];
-    NSString *preview = [note.content length] == 0 ? @"New note..." : [note.content stringByGeneratingPreview];
+    NSColor *headlineColor = _highlighted ? [self.theme colorForKey:@"tintColor"] : [self.theme colorForKey:@"noteHeadlineFontColor"];
+    NSColor *previewColor = _highlighted ? [self.theme colorForKey:@"tintColor"] : [self.theme colorForKey:@"noteBodyFontPreviewColor"];
+    NSString *preview = [_note.content length] == 0 ? @"New note..." : [_note.content stringByGeneratingPreview];
```

Comment:
I know this was existing code, but do you think we should use `NSLocalizedString` for `@"New note..."`?

Reply:
ARGH didn't notice!!! of course we can, apologies abiout missing that!!